static/styles/dark_theme.css

```diff
@@ -234,8 +234,7 @@ body.dark-theme {
     :disabled,
     input:not([type="radio"]):read-only,
     textarea:read-only,
-    .sidebar-title,
-    .recipient_row_date {
+    .sidebar-title {
```

Comment:
I think this hunk needs to be removed, if this commit is just extracting variables?